app/src/main/res/layout/fragment_profile.xml

```diff
@@ -14,32 +14,33 @@
         android:layout_height="match_parent"
         tools:context=".fragments.ProfileFragment">
 
-            <ImageView
-                android:id="@+id/ivUserProfile"
-                android:layout_width="100dp"
-                android:layout_height="100dp"
-                android:layout_marginTop="20dp"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintStart_toStartOf="parent"
-                app:layout_constraintTop_toTopOf="parent"
-                tools:src="@tools:sample/avatars" />
+        <ImageView
+            android:id="@+id/ivUserProfile"
+            android:layout_width="@dimen/image_width_height"
+            android:layout_height="@dimen/image_width_height"
+            android:layout_marginTop="@dimen/comic_list_margins"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.498"
```

Comment:
just wondering, how does this change the UI?

Reply:
I don't think it does, I think it may have been generated when I was using design view and I just didn't remove it. I have removed it and my UI remains the same